### PR TITLE
Fixed phpdoc

### PR DESCRIPTION
**Pull request by core23**

### CODE REVIEW

File: Twig/SecurityPolicyContainerAware.php [hunk 1]
```diff
@@ -24,23 +25,44 @@
  */
 class SecurityPolicyContainerAware implements \Twig_Sandbox_SecurityPolicyInterface
 {
+    /**
+     * @var
```
Comment (OskarStark) [inline Twig/SecurityPolicyContainerAware.php]: ping @core23

**Comment by OskarStark** [comment]
after fixing my comment, this is RTM

Oskar

**Comment by core23** [comment]
Fixed @OskarStark

File: Formatter/FormatterInterface.php [hunk 2]
```diff
@@ -16,23 +16,17 @@
 interface FormatterInterface
 {
     /**
-     * @abstract
```
Comment (Soullivaneuh) [inline Formatter/FormatterInterface.php]: Not sure for that. cc @rande

Comment (core23) [inline Formatter/FormatterInterface.php]: @rande Can you say something about the ``@abstract`` annotation? Could I remove this or should it be keept on interfaces?

Comment (rande) [inline Formatter/FormatterInterface.php]: You could remove it. It is a mistake.

**Comment by core23** [comment]
RTM @OskarStark

**Comment by OskarStark** [comment]
Thank you @core23

**Pull request merged**